src/main/java/org/oscii/api/LexiconProtocol.java

```diff
@@ -42,6 +52,9 @@ public Response respond(Request request) {
         if (request.example) addExamples(request, response);
         if (request.extend) addExtensions(request, response);
         if (request.synonym) addSynonyms(request, response);
+        if (request.wordvec) addWordVector(request, response);
```

Comment:
Also, there are embeddings of whole phrases (not just words) in the Google News corpus, right? Do we support looking those up?